code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2502,9 +2499,9 @@
 	..()
 	T.add_nutrientlevel(1)
 	if(prob(25*custom_plant_metabolism))
-		T.add_weedlevel(3)
+		T.add_weedlevel(10)
```

Comment:
Haven't you decreased this to 3 twice already? Like in #32816. Why return it to 10?

Reply:
Yeah, because the rates were messed up due to the lines I am removing now. Do you want to keep it at 3?

Reply:
Dunno how the changes affect it, I'm just curious as to why the 10-3-10-3-10 cycle.

Reply:
10 should be the amount that should make it consistent with the old hydroponics